### PR TITLE
1.4.0 version bump + changelog

### PR DESCRIPTION
**Pull request by douglasmuraoka**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
     "Cloud Code Viewer",
     "Cloud Code Jobs Viewer and Runner",
     "Parse Config",
-    "API Console",
+    "REST API Console",
+    "GraphQL API Console",
```
Comment (acinader) [inline package.json]: niiiiice

**Comment by acinader** [comment]
alright, so this is now in master.  you can create and publish the release whenever you're ready.

**Comment by acinader** [comment]
getting so close to my drop-down app switcher working again, I can almost taste it!

**Comment by douglasmuraoka** [comment]
@acinader I think I don't have permission to create a release draft...

**Comment by acinader** [comment]
fixed.  try again...

**Comment by douglasmuraoka** [comment]
@acinader @TomWFox done!! :)

**Comment by acinader** [comment]
unfortunately our NPM deploy failed.  I know why and I will fix now.

**Comment by acinader** [comment]
w000000!  my dropdown works again.  Much thanks @douglasmuraoka

**Pull request merged**